Templates/settings.blade.php

```diff
@@ -2,14 +2,31 @@
 
 @section('content')
 
-<x-global::pageheader :icon="'fa fa-puzzle-piece'">
-    <h1>TimeTable Settings</h1>
-</x-global::pageheader>
+    <x-global::pageheader :icon="'fa fa-puzzle-piece'">
+        <h1>TimeTable Settings</h1>
+    </x-global::pageheader>
 
 
-<div class="maincontent">
-    <div class="maincontentinner">
-        <h1>Hello World settings!</h1>
+    <div class="maincontent">
+        <?php if (isset($tpl)) {echo $tpl->displayNotification();}  ?>
+        <div class="maincontentinner">
+            <h5 class="subtitle">TimeTable Settings</h5>
+            <p style="padding-bottom: 15px">These settings will change the way the TimeTable plugin works.</p>
```

Comment:
Not a "must fix" but.... 
Not a huge fan of inline styling or `px` values. Looks like Leantime is using bootstrap (?), so look at https://getbootstrap.com/docs/5.3/utilities/spacing/#margin-and-padding or add a class in the plugins css file